### PR TITLE
add release template description

### PR DESCRIPTION
**Pull request by laisbsc**
Add REALEASE.md document with descriptions.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -7,13 +7,35 @@
 2. Ensure that the version number in `qhub/VERSION` is used in pinning QHub in the github actions 
 `qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-config.yaml`
 ---
-## Upcoming release 0.1.22
-### Features changes and enhancements
-* 
+## Release 0.2.0
+### Feature changes and enhancements
```
Comment (aktech) [inline RELEASE.md:11]: maybe comma after Features?

File: RELEASE.md [hunk 2]
```diff
@@ -7,13 +7,35 @@
...
+* Added conda prerequisites for GUI packages.
+* Added `qhub destroy` functionality that tears down the QHub deployment.
+* Changed the default repository name from `master` to `main`.
```
Comment (aktech) [inline RELEASE.md:14]: repository name -> repository branch

File: RELEASE.md (hunk 1 above)
Comment (laisbsc) [reply RELEASE.md:11]: sure :)

**Comment by aktech** [comment]
Looks good apart from a couple of nitpicks.

File: RELEASE.md (hunk 1 above)
Comment (laisbsc) [inline RELEASE.md:11]:
```suggestion
### Feature changes, and enhancements
```

File: RELEASE.md (hunk 2 above)
Comment (laisbsc) [inline RELEASE.md:14]:
```suggestion
* Changed the default repository branch from `master` to `main`.
```

**Pull request merged**